etc/examples/nodes/miob.conf

```diff
@@ -0,0 +1,55 @@
+logging = {
+    level = "debug"
+}
+
+nodes = {
+    fpga_0 = {
+        type = "fpga",
+        card = {
+            interface = "platform",
+            id = "",
+            slot = "",
+            do_reset = false,
+            ips = "/home/root/codebase/node/etc/fpga/miob/ips.json",
```

Comment:
Can this be specified by a relative path? Having a hardcoded absolute path here isnt nice.